translations/base-fi.yaml

```diff
@@ -662,82 +651,75 @@ storyRewards:
     reward_blueprints:
         title: Piirustukset
         desc: Nyt voit <strong>Kopioida ja Liittää</strong> paloja tehtaastasi! Valitse
-            alue (pidä pohjassa CTRL ja raahaa hiirellä), ja paina 'C'
-            kopioidaksesi.<br><br>Sen liittäminen ei ole
-            <strong>ilmaista</strong>, sinun täytyy tuottaa <strong>piirustus
-            muotoja</strong> jotta sinulla on varaa siihen! (Ne mitkä juuri
-            toimitit).
+            alue (pidä CTRL pohjassa ja raahaa hiirellä), ja paina 'C'
+            kopioidaksesi piirrustuksen.<br><br>Sen liittäminen ei ole
+            <strong>ilmaista</strong>. Sinun täytyy tuottaa <strong>piirustusmuotoja</strong>,
+            jotta sinulla on varaa siihen! (Ne mitkä juuri toimitit).
     no_reward:
         title: Seuraava taso
-        desc: "This level gave you no reward, but the next one will! <br><br> PS: Better
-            don't destroy your existing factory - You need <strong>all</strong>
-            those shapes later again to <strong>unlock upgrades</strong>!"
+        desc: Et saanut palkintoa tältä tasolta, mutta seuraavalta tasolta saat! <br><br> PS: Parempi
```

Comment:
Put everything after `desc: ` in quotation marks `"`

Reply:
The whole paragraph

Reply:
Like in line 317ff. `:` colon is a key that makes the lint fail if you dont quote the whole translation

Reply:
noticed that.